test_scripts/iAP2TransportSwitch/common.lua

```diff
@@ -94,6 +93,25 @@ function module:connectMobile(pDevice)
   return module:expectEvent(events.connectedEvent, "Connected", pDevice)
 end
 
+function module:doTransportSwitch(device)
+  input = io.open(config.pathToSDL.."/iap_signals_in", "w")
```

Comment:
@dev-gh Please use local variable here and below

Reply:
@dboltovskyi 
Done with 24aad0b